docroot/css/app.scss

```diff
@@ -30,6 +30,24 @@
     margin-top: 15px;
   }
 }
+#map-canvas {
+  height: 500px;
+  width:960px;
+  margin: 0px;
+  padding: 0px
+}
+img[src="public/images/circle.gif"] {
+  background:#205493;
```

Comment:
Just curious, but doesn't this background color change by the provider type? would it make sense to add classes for these images?